### PR TITLE
App 2019

### PR DESCRIPTION
**Pull request by lakshaycoder01**
review this branch

### CODE REVIEW

File: content/eng/manifesto/aap_2019.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+draft: false
+---
+
+**AAP** has released their **manifesto** for the **General Elections of 2019** which is happening in 7 phases between 11 April to 19 May.
+
+Here, I will note down some of policies and goals **AAP** has led down for their 2019 **General Elections**.
+
+1.  #### FULL STATEHOOD FOR DELHI
+
+    1.  ### Land
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: put 5 # instead of 3 for sub heading

File: content/eng/manifesto/aap_2019.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+        a. Allocate land to build hundreds of Schools and Colleges will ensure:
+
+            - Free, guaranteed schooling facilities for all children in the age group of 3 to 18 years(nursery to 12th) residing in Delhi
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: full stop in end

File: content/eng/manifesto/aap_2019.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            - All students securing above 60% marks in class 12 from Delhi schools will be guaranteed admission in a college.
+
+        b. Build new Mohalla clinics to provide for Universal Healthcare at the doorstep(within 1km) of each citizen.
+        c. Expansion of existing hospitals and build new hospitals to create provision for thousand of additional beds.
+        d. Every single family in Delhi will be able to own their own house in 10 years. '
+        e. Additional bus depots and terminals to cater to all the extra buses that Delhi needs.
+        f. Ensure a congestion free Delhi by removal of road encroachments and improving road design.
+
+    2.  ### Law and Order
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: same 5 #

File: content/eng/manifesto/aap_2019.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+        a. Take all steps necessary to make Delhi the safest city in India for women. All cases of crime against woman to be tried by Fast Track courts.
+        b. Improve the conviction rate for all crimes in Delhi, especially against women which is currently at 19%
+        c. Ensure higher and equitable distribution of PCR across Delhi so that the response time of a PCR to reach a caller anywhere in Delhi is less than 5 minutes.
+        d. Revive the Anti-Corruption Bureau(ACB) just like we did in the 49 days government to break the corruption in Delhi.
+
+    3.  ### Police
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: same 5 #

File: content/eng/manifesto/aap_2019.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+        a. Make Delhi Police accountable to the people of Delhi. All ground personnel will be equipped with Police Body Cameras and CCTVs will be installed in all corners of police stations.
+        b. Fill nearly 2/3rd of the posts vacant in Delhi Police and ensure reservation of 33% of the posts for women.
+        c. Professionalization of police force by providing world-class skills training on advance methods of policing and modern infrastructure.
+        d. Initiate long-pending police reforms as directed by the Hon'ble Supreme Court in the matter of Prakash Singh versus Union of India,2006.
+
+    4.  ### Control Over Officials and Decision Making
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: use 5 #

File: content/eng/manifesto/aap_2019.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    2. Students of Delhi Schools securing more than 60% marks in Class 12 shall get guaranteed admission in a college as a regular student.
+    3. In every college of Delhi, 85% seats will be reserved for the residents of Delhi.
+    4. Build new schools and colleges and recruit regular teachers as per the requirement to provide quality education in accordance with new law.
+    5. All existing Guest teachers serving the schools of Delhi government shall be regularized.
+    6. New Medical and Engineering colleges shall be started to increase the annual intake by 10,000.
+    7. A 200 point roster will be implemented for higher education institutes(Universities and Colleges).
+
+These are the promises that AAP made in general election of 2019.
+
+If you want to read what else promised by AAP in year 2019, please go through [their manifesto][1].
```
Comment (Abby3017) [inline content/eng/manifesto/aap_2019.md]: dont use please

**Review by Abby3017 — CHANGES_REQUESTED** [review]

**Review by lakshaycoder01 — COMMENTED** [review]
Done

**Review by Abby3017 — APPROVED** [review]

**Pull request merged**